Core/Src/compute.c

```diff
@@ -14,14 +16,22 @@ enum { CHARGE_ENABLED, CHARGE_DISABLED };
 extern CAN_HandleTypeDef hcan1;
 extern CAN_HandleTypeDef hcan2;
 
+extern TIM_HandleTypeDef htim1;
+extern TIM_HandleTypeDef htim8;
+
+TIM_OC_InitTypeDef pwm_config;
+
+const uint32_t fan_channels[6] = {TIM_CHANNEL_3, TIM_CHANNEL_1, TIM_CHANNEL_4, TIM_CHANNEL_3, TIM_CHANNEL_2, TIM_CHANNEL_1};
```

Comment:
Nit: Can you use `FANMAX` to declare the size of this array, doing so indicates you can index by the enum